src/stretch/agent/robot_agent.py

```diff
@@ -139,8 +140,21 @@ def __init__(
         if parameters["motion_planner"]["simplify_plans"]:
             self.planner = SimplifyXYT(self.planner, min_step=0.05, max_step=1.0, num_steps=8)
 
+        # Map updates
+        self._robot_lock = Lock()
+        self._update_map_thread = Thread(target=self.update_map_loop)
+        self._update_map_thread.start()
+
+        # Get observations thread
+        self._robot_lock = Lock()
```

Comment:
we should have a flag, and default this to false, at least initially

we can switch it to default true after more testing + making sure the apps all run properly